spel/minimal-linux.pkr.hcl

```diff
@@ -476,6 +476,18 @@ variable "amigen7_storage_layout" {
 # Variables used by AMIgen8
 ###
 
+variable "amigen8_extra_rpms" {
+  description = "List of package specs (rpm names or URLs to .rpm files) to install to the EL8 builders and images"
+  type        = list(string)
+  default = [
+    "python38",
```

Comment:
Per the [Red Hat document](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_installing-and-using-python_configuring-basic-system-settings): 

> In Red Hat Enterprise Linux 8, Python 3 is distributed in versions 3.6, 3.8, and 3.9, provided by the python36, python38, and python39 modules in the AppStream repository.

Do we maybe want to be proactive and install Python 3.9 instead of 3.8?

Reply:
Ok, updated, please take a look again!